index.js

```diff
@@ -20,6 +21,17 @@ mongoose.connect(
   },
   (e) => {
     console.log("db connection", !e ? "successfull" : e);
+    if (!e) {
+      Currency.countDocuments({}, function (err, count) {
+        if (err) console.log(err);
+        else {
+          if (!count) {
```

Comment:
What will happen if I add a new currency in the list later on, will it still be inserted in the DB since you are counting the documents and than checking if there is at least one entry in the DB and calling createCurrency() only if !count

Reply:
I don't think I understand exactly what you are trying to say. With the code that's here, a new currency can be added later on. Should this not be possible?

Reply:
Ignore me, I forgot about the fact that you need to implement the seeding for the DB. Now it makes sense for me.